pkg/services/notification/am.go

```diff
@@ -7,18 +7,94 @@ package notification
 import (
 	"context"
 
+	"openpitrix.io/notification/pkg/models"
+
 	"openpitrix.io/notification/pkg/constants"
 	"openpitrix.io/notification/pkg/manager"
 	"openpitrix.io/notification/pkg/pb"
 )
 
+var NfStatuses = []string{
+	constants.StatusPending,
+	constants.StatusSending,
+	constants.StatusSuccessful,
+	constants.StatusFailed,
+}
+
+var NfTypes = []string{
+	constants.NotifyTypeEmail,
+	constants.NotifyTypeWeb,
+	constants.NotifyTypeMobile,
+	constants.NotifyTypeSms,
+	constants.NotifyTypeWeChat,
+}
+
+var Statuses = []string{
+	constants.StatusActive,
+	constants.StatusDisabled,
+	constants.StatusDeleted,
+}
+
+var ContentTypes = []string{
+	constants.ContentTypeInvite,
+	constants.ContentTypeverify,
+	constants.ContentTypeFee,
+	constants.ContentTypeBusiness,
+	constants.ContentTypeOther,
+}
+
 func (s *Server) Checker(ctx context.Context, req interface{}) error {
 	switch r := req.(type) {
 	case *pb.ServiceConfig:
 		return manager.NewChecker(ctx, r).
 			Required(constants.ServiceCfgProtocol, constants.ServiceCfgEmailHost, constants.ServiceCfgPort, constants.ServiceCfgDisplayEmail, constants.ServiceCfgEmail, constants.ServiceCfgPassword).
 			Exec()
+	case *pb.GetServiceConfigRequest:
+		return manager.NewChecker(ctx, r).
+			StringChosen("service_type", NfTypes).
```

Comment:
You can change all of the parameters into `constants.xxx`

Reply:
updated.